NERODevelopment/src/controllers/signalscontroler.cpp

```diff
@@ -0,0 +1,33 @@
+#include "signalscontroler.h"
+
+signalsController::signalsController(QObject *parent)
+{
+    // Inital mockup of statuses, Strings represent
+    // the name and status of each attribute
+    attributeStatus["SideBRBs"] = "unchecked";
+    attributeStatus["BMS"] = "unchecked";
+    attributeStatus["IMD"] = "unchecked";
+    attributeStatus["BSPD"] = "unchecked";
+    attributeStatus["MPU"] = "unchecked";
+    attributeStatus["BOTS"] = "unchecked";
+    attributeStatus["INERTIA"] = "unchecked";
```

Comment:
also the unchecked should be an enumeration of possible values, so like good, faulted, off etc

Reply:
you can do that through macros in a file

Reply:
Done!